telemetry/definitions/commonDefinitions.json

```diff
@@ -3882,7 +3884,7 @@
         {
             "name": "codeTransform_logGeneralError",
             "passive": true,
-            "description": "Deprecated - avoid logging every service call",
+            "description": "Emit general error that occurred during the transformation.",
```

Comment:
this line has conflict with https://github.com/aws/aws-toolkit-common/pull/797/files PR.

Reply:
Reverted. Will let the change in #797 go through.